Fix Case-sensitive directory preservation in project reconstruction #27

Mabbs opened the pull request.

Mabbs committed:
fix Case-sensitive directory preservation in project reconstruction #27

Mabbs committed:
Update ProjectReconstructor.cs

uuksu commented on RPGMakerDecrypter.MVMZ/ProjectReconstructor.cs:
Please use English only to keep the code consistent.

I prefer also using full variable names instead of abbreviations. For example `sourceDirs` -> `sourceDirectories`.

uuksu commented:
Hey,

The given solution looks good, please fix few quality issues and I will approve this.

Mabbs commented:
> Hey,
> 
> The given solution looks good, please fix few quality issues and I will approve this.

I apologize, but I'm not proficient in C#. The code modifications were AI-generated. Please let me know if you'd like me to clarify anything about the changes.

uuksu commented:
I'm sorry, I forgot to publish the review. Here are the changes for the PR I request.

It's funny that my AI-sense was tickling and you confirmed it :)

Mabbs committed:
Refactor variable names to use full words for clarity

Mabbs commented:
> I'm sorry, I forgot to publish the review. Here are the changes for the PR I request.
> 
> It's funny that my AI-sense was tickling and you confirmed it :)

Sure, I've addressed the issues in the latest commit.

uuksu commented on RPGMakerDecrypter.MVMZ/ProjectReconstructor.cs:
```diff
@@ -10,6 +10,7 @@ public abstract class ProjectReconstructor
             "audio",
             "css",
             "data",
+            "dataex",
```
Have you confirmed by yourself that there actually can be a directory with this name?

Mabbs replied:
Yes, I found that many games have this directory

Mabbs replied:
> Have you confirmed by yourself that there actually can be a directory with this name?

<img width="649" height="417" alt="image" src="https://github.com/user-attachments/assets/ac13f7b5-6bd3-46e2-8a20-99b416e9bd95" />
